### PR TITLE
docs(cn): translate src/content/api/compilation-object.md

### PR DESCRIPTION
**Pull request by jacob-lcs**
_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by jacob-lcs** [comment]
@QC-L

File: src/content/api/compilation-object.md [hunk 1]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 
-## compilation object methods
+## compilation 对象方法
 
 ### getStats
 
 `function`
 
-Returns Stats object for the current compilation.
+返回当前编译(compilation)的状态对象。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:19]:
```suggestion
返回当前编译的状态对象。
```

Comment (QC-L) [reply src/content/api/compilation-object.md:19]: 没必要修饰，如果修饰，一个地方添加即可，后面可省略。

File: src/content/api/compilation-object.md [hunk 2]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 
 ### addModule
 
 `function (module, cacheGroup)`
 
-Adds a module to the current compilation.
+向当前编译(compilation)添加一个模块。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:25]: 同上

File: src/content/api/compilation-object.md [hunk 3]
```diff
@@ -8,261 +8,261 @@ contributors:
...
-- `cacheGroup` - `cacheGroup` of the module
+- `module` - 要添加的模块
+- `cacheGroup` - 模块的 `cacheGroup`
 
 ### getModule
 
 `function (module)`
 
-Fetches a module from a compilation by its identifier.
+通过编译(compilation)的标识符获取其模块。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:36]: 同上

File: src/content/api/compilation-object.md [hunk 4]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 `function (module)`
 
-Attempts to search for a module by its identifier.
+尝试通过其标识符搜索模块。
 
-Parameters:
+参数：
 
-- `module` - module to be searched for. The identifier is extracted from the module by the compilation using `module.identifier()` method.
+- `module` - 要搜索的模块。标识符是使用通过编译(compilation)使用 `module.identifier()` 方法从模块中提取的。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:50]: 此句不通顺，建议阅读一下，修改下

Comment (QC-L) [reply src/content/api/compilation-object.md:50]: 一句话出现了两个使用

File: src/content/api/compilation-object.md [hunk 5]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 
-- `module` - module to add dependencies to.
-- `dependencies` - set of sorted dependencies to iterate through and add to the module.
-- `bail` - whether to bail or not when an error occurs.
-- `cacheGroup` - `cacheGroup` of the module.
-- `recursive` - whether it is a recursive traversal.
-- `callback` - function to invoke after adding the module dependencies.
+- `module` - 要添加依赖的模块。
+- `dependencies` - 要遍历并添加到模块的一组已排序依赖。
+- `bail` - 发生错误时是否中断进程并抛出失败结果。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:97]:
```suggestion
- `bail` - 发生错误时是否中断进程并抛出 error。
```

File: src/content/api/compilation-object.md [hunk 6]
```diff
@@ -8,261 +8,261 @@ contributors:
...
+参数：
 
-- `module` - module to add dependencies to.
-- `dependencies` - set of sorted dependencies to iterate through and add to the module.
-- `bail` - whether to bail or not when an error occurs.
-- `cacheGroup` - `cacheGroup` of the module.
-- `recursive` - whether it is a recursive traversal.
-- `callback` - function to invoke after adding the module dependencies.
+- `module` - 要添加依赖的模块。
+- `dependencies` - 要遍历并添加到模块的一组已排序依赖。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:96]:
```suggestion
- `dependencies` - 需遍历添加到模块的一组已排序依赖。
```

File: src/content/api/compilation-object.md [hunk 7]
```diff
@@ -8,261 +8,261 @@ contributors:
...
+- `recursive` - 是否要递归遍历。
+- `callback` - 添加模块依赖之后回调的函数。
 
 
 ### addEntry
 
 `function (context, entry, name, callback)`
 
-Adds an entry to the compilation.
+为编译(compilation)添加入口。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:107]: 同上

File: src/content/api/compilation-object.md [hunk 8]
```diff
@@ -8,261 +8,261 @@ contributors:
...
-- `thisCallback` - function to be invoked when the module finishes rebuilding.
+- `module` - 要被重建的模块。
+- `thisCallback` - 模块重建完成之后调用的函数。
 
 ### finish
 
 `function (callback)`
 
-Finishes compilation and invokes the given callback.
+完成编译(compilation)并调用给定的回调。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:143]: 同上

File: src/content/api/compilation-object.md [hunk 9]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 
-Parameters:
+参数：
 
-- `callback` - function to be invoked when the compilation has been finished.
+- `callback` - 编译(compilation)完成之后调用的函数。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:147]: 同上

File: src/content/api/compilation-object.md [hunk 10]
```diff
@@ -8,261 +8,261 @@ contributors:
...
 
 ### seal
 
 `function (callback)`
 
-Seals the compilation.
+封闭编译(compilation)。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:153]: 同上

File: src/content/api/compilation-object.md [hunk 11]
```diff
@@ -8,261 +8,261 @@ contributors:
...
-- `dependency` - the dependency to get reference to.
+- `module` - 有问题的模块。
+- `dependency` - 要引用的依赖。
 
 ### processDependenciesBlocksForChunkGroups
 
 `function (inputChunkGroups)`
 
-Creates the `Chunk` graph from the `Module` graph. The process is done in two phases. Phase one: traverse the module graph and build a basic chunks graph in `chunkDependencies`. Phase two: traverse every possible way through the basic chunk graph and track the available modules. While traversing, `processDependenciesBlocksForChunkGroups` connects chunks with each other and `Blocks` with `Chunks`. It stops traversing when all modules for a chunk are already available and it doesn't connect unneeded chunks.
+从 `Module` 图创建 `Chunk` 图。该过程分为两个阶段完成。阶段一：遍历模块图，在 `chunkDependencies` 中创建一个基础 chunks 图。阶段二：通过基本 chunks 图遍历所有可能的方法并且跟踪可用模块。遍历过程中 `processDependenciesBlocksForChunkGroups` 将 chunk 相互连接并且`Blocks` 与 `Chunks` 连接. 当一个 chunk 的所有模块都已经可用并且不连接不需要的 chunk 时，它将停止遍历。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:228]:
```suggestion
从 `Module` 图创建 `Chunk` 图。该过程分为两个阶段完成。阶段一：遍历模块图，在 `chunkDependencies` 中创建一个基础 chunk 图。阶段二：通过基本 chunk 图遍历所有可能的方法并且跟踪可用模块。遍历过程中 `processDependenciesBlocksForChunkGroups` 将 chunk 相互连接并且 `Blocks` 与 `Chunks` 连接. 当一个 chunk 的所有模块都已经可用并且不连接不需要的 chunk 时，它将停止遍历。
```

File: src/content/api/compilation-object.md [hunk 12]
```diff
@@ -8,261 +8,261 @@ contributors:
...
-- `chunk` - a chunk to patch tie.
+- `module` - 要修复关系的模块。
+- `chunk` - 要修复关系的 chunk。
 
 ### removeChunkFromDependencies
 
 `function (block, chunk)`
 
-Removes given chunk from a dependencies block module and chunks after removing dependency reasons. Called automatically by `removeReasonsOfDependencyBlock`.
+在除去依赖性原因后，从依赖块模块和 chunks 中移除给定的 chunk。被 `removeReasonsOfDependencyBlock` 自动调用。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:260]: chunks 无需复数，中文中的英文不需要 s

Comment (QC-L) [inline src/content/api/compilation-object.md:260]:
```suggestion
在除去依赖性原因后，从依赖块模块和 chunks 中移除给定的 chunk。会被 `removeReasonsOfDependencyBlock` 自动调用。
```

File: src/content/api/compilation-object.md [hunk 13]
```diff
@@ -296,35 +296,35 @@ Parameters:
...
-Allows running another instance of webpack inside of webpack. However, as a child with different settings and configurations applied. It copies all hooks and plugins from the parent (or top-level compiler) and creates a child `Compiler` instance. Returns the created `Compiler`.
+允许在webpack中运行另一个webpack实例。但是，子编译器会应用不同的设置和配置。他会从父编译器（或者顶级编译器）中复制所有的钩子(hooks)和插件(plugins)，并且创建一个子 `Compiler`实例。 返回创建好的 `Compiler`。
 
-Parameters:
+参数：
 
-- `name` - name for the child `Compiler`.
-- `outputOptions` - output options object.
-- `plugins` - webpack plugins that will be applied.
+- `name` -  子 `Compiler`的名称。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:325]:
```suggestion
- `name` -  子 `Compiler` 的名称。
```

File: src/content/api/compilation-object.md [hunk 14]
```diff
@@ -296,35 +296,35 @@ Parameters:
...
-- `data` - data object.
+- `filename` - 用于通过哈希获取资源路径。
+- `data` - 数据对象。
 
 ### createChildCompiler
 
 `function (name, outputOptions, plugins)`
 
-Allows running another instance of webpack inside of webpack. However, as a child with different settings and configurations applied. It copies all hooks and plugins from the parent (or top-level compiler) and creates a child `Compiler` instance. Returns the created `Compiler`.
+允许在webpack中运行另一个webpack实例。但是，子编译器会应用不同的设置和配置。他会从父编译器（或者顶级编译器）中复制所有的钩子(hooks)和插件(plugins)，并且创建一个子 `Compiler`实例。 返回创建好的 `Compiler`。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:321]:
```suggestion
允许在 webpack 中运行另一个 webpack 实例。但是，子编译器会应用不同的设置和配置。他会从父编译器（或者顶级编译器）中复制所有的钩子(hook)和插件(plugin)，并且创建一个子 `Compiler`实例。返回值为创建好的 `Compiler` 实例。
```

File: src/content/api/compilation-object.md [hunk 15]
```diff
@@ -334,40 +334,40 @@ Parameters:
 
 `function (file, source, assetInfo = {})`
 
-W> Available since webpack 4.40.0
+W> webpack 4.40.0 后是可用的。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:337]:
```suggestion
W> webpack 4.40.0 后可用。
```

File: src/content/api/compilation-object.md [hunk 16]
```diff
@@ -334,40 +334,40 @@ Parameters:
...
+- `file` - 资源名称。
+- `source` - 资源来源。
+- `assetInfo` - 附加资源信息。
 
 ### updateAsset
 
 `function (file, newSourceOrFunction, assetInfoUpdateOrFunction)`
 
-W> Available since webpack 4.40.0
+W> webpack 4.40.0 后是可用的。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:349]:
```suggestion
W> webpack 4.40.0 后可用。
```

File: src/content/api/compilation-object.md [hunk 17]
```diff
@@ -334,40 +334,40 @@ Parameters:
...
+- `file` - 资源名称。
+- `newSourceOrFunction` - 新资源来源或将旧资源转换为新资源的函数。
+- `assetInfoUpdateOrFunction` - 新资源信息或将旧资源转换为新资源的函数。
 
 ### getAssets
 
 `function`
 
-W> Available since webpack 4.40.0
+W> webpack 4.40.0 后是可用的。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:361]:
```suggestion
W> webpack 4.40.0 后可用。
```

File: src/content/api/compilation-object.md [hunk 18]
```diff
@@ -334,40 +334,40 @@ Parameters:
...
 
-Returns array of all assets under the current compilation.
+返回当前编译(compilation)下所有资源的数组。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:363]: 同上

File: src/content/api/compilation-object.md [hunk 19]
```diff
@@ -334,40 +334,40 @@ Parameters:
...
 
 ### getAsset
 
 `function (name)`
 
-W> Available since webpack 4.40.0
+W> webpack 4.40.0 后是可用的。
```
Comment (QC-L) [inline src/content/api/compilation-object.md:369]:
```suggestion
W> webpack 4.40.0 后可用。
```

Comment (jacob-lcs) [reply src/content/api/compilation-object.md]: :heavy_check_mark:已做出对应修改

**Pull request merged**